_pages/documentation.md

```diff
@@ -1,12 +0,0 @@
----
-title: Documentation
```

Comment:
## Delete: documentation page
Deleted this as we will have documentation for each project and the server separately. Although since the server is our main product, I can add it back for that purpose if people want.

I also commented out the `docs` section in `navigation.yml`. I'm assuming that the way this page is set up works in tandem with that yaml.